src/main/java/com/box/sdk/BoxMetadataCascadePolicy.java

```diff
@@ -0,0 +1,255 @@
+                        new BoxMetadataCascadePolicy(api, jsonObject.get("id").asString());
+
+                return cascadePolicy.new Info(jsonObject);
+            }
+        };
+    }
+
+    /**
+     * Returns the information for a specific BoxMetadataCascadePolicy.
+     *
+     * @param fields the fields to retrieve.
+     * @return the information about this metadata cascade policy.
+     */
+    public BoxMetadataCascadePolicy.Info getInfo(String... fields) {
+        QueryStringBuilder builder = new QueryStringBuilder();
+        if (fields.length > 0) {
+            builder.appendParam("fields", fields);
+        }
+        URL url = METADATA_CASCADE_POLICIES_URL_TEMPLATE.buildWithQuery(this.getAPI().getBaseURL(),
+                builder.toString(), this.getID());
+        BoxAPIRequest request = new BoxAPIRequest(this.getAPI(), url, "GET");
+        BoxJSONResponse response = (BoxJSONResponse) request.send();
+        JsonObject responseJSON = JsonObject.readFrom(response.getJSON());
+        return new Info(responseJSON);
+    }
+
+    /**
+     * Creates a new Metadata Cascade Policy on a folder.
+     *
+     * @param api         the API connection to be used by the resource.
+     * @param folderID    the ID of the folder to create a metadata cascade policy on.
+     * @param scope       the scope of the metadata cascade policy.
+     * @param templateKey the key of the template.
+     * @return information about the Metadata Cascade Policy.
+     */
+    public static BoxMetadataCascadePolicy.Info create(final BoxAPIConnection api, String folderID, String scope,
+                                                       String templateKey) {
+        URL url = GET_ALL_METADATA_CASCADE_POLICIES_URL_TEMPLATE.build(api.getBaseURL());
+        BoxJSONRequest request = new BoxJSONRequest(api, url, "POST");
+        JsonObject requestJSON = new JsonObject()
+                .add("folder_id", folderID)
+                .add("scope", scope)
+                .add("templateKey", templateKey);
+        request.setBody(requestJSON.toString());
+        BoxJSONResponse response = (BoxJSONResponse) request.send();
+        JsonObject responseJSON = JsonObject.readFrom(response.getJSON());
+        BoxMetadataCascadePolicy createdMetadataCascadePolicy = new BoxMetadataCascadePolicy(api,
+                responseJSON.get("id").asString());
+        return createdMetadataCascadePolicy.new Info(responseJSON);
+    }
+
+    /**
+     * If a policy already exists on a folder, this will apply that policy to all existing files and sub folders within
+     * the target folder.
+     *
+     * @param api                the API connection to be used by the resource.
+     * @param conflictResolution the desired behavior for conflict-resolution. Set to either none or overwrite.
+     * @param cascadePolicyID    the ID of the metadata cascade policy.
+     */
+    public static void forceApply(final BoxAPIConnection api, String conflictResolution,
```

Comment:
This should probably be an instance method, not `static`, since it uses the ID of a specific cascade policy.  That way it wouldn't need to take in the ID as a parameter

Reply:
Remember to make this non-static!